src/main/haskell/language-kore/src/Data/Kore/Step/StepperAttributes.hs

```diff
@@ -0,0 +1,70 @@
+{-|
```

Comment:
This file seems to be missing some things from testing, could you fix?

Reply:
Thanks for pointing it out. I forgot to add the test file to Git :-)